docs/source/custom/installation.rst

```diff
@@ -157,15 +157,30 @@ it is likely you will need to specify a legacy compiling configuration by changi
 
 2. Configure your environment by pointing the SIXS_DIR variable to point to your installation directory.
 
-3. Download the `pre-trained sRTMnet neural network <https://zenodo.org/record/4096627>`_, and (for the example below)
-point the environment variable EMULATOR_PATH to the base unzipped path.
+3. Download the `pre-trained sRTMnet neural network <https://avng.jpl.nasa.gov/pub/PBrodrick/isofit/sRTMnet_v100.h5>`_,
+as well as some `auxiliary data <https://avng.jpl.nasa.gov/pub/PBrodrick/isofit/sRTMnet_v100_aux.npz>`_.
+This will give you an hdf5 and an aux file. It is important that you store both in the same directory.
```

Comment:
Could you changes these to reference the v120 versions?

Reply:
Sure! Should I update it throughout the entire codebase or just in the docs?

Reply:
Let's do it everywhere if you don't mind.  It's our current operational model, so best to point people in the right place :-).